ui/StatusQ/src/StatusQ/Components/StatusEmojiAndColorComboBox.qml

```diff
@@ -38,13 +38,13 @@ StatusComboBox {
     id: root
 
     /*!
-       \qmlproperty string StatusEmojiAndColorComboBox::v
+       \qmlproperty string StatusEmojiAndColorComboBox::defaultAssetName
```

Comment:
Ohh!! Thanks for fixing that! I think that this was my fault copy&pasting :grimacing: